### ISSUE TITLE
Improve the ‘manage your team / users’ userflow

### ISSUE THREAD

**Issue by lusebille** [gristlabs/grist-core#1304]
Looking this feedback and other users feebacks  about it , it will be helpful to be able to find easily a user using search or sorting when you'be got lot of members in your team ( can be applied to workspaces and documents too ) https://community.getgrist.com/t/quite-hard-to-find-specific-team-emails/7163

**Comment by lusebille** [comment]
to discuss https://www.figma.com/design/wcpetFt6aOKzTszcvPPWLQ/%5B05%2F24%5D-Grist-Design?node-id=3082-103250&t=KmV2MAenZUNU3Plk-1

**Comment by ouvry-ems** [comment]
Totally support that idea!

Also I'd like to add the fact that currently the user list is not sorted by access rules. It would be great to quickly see who has the most rights, from owners to editors to viewers. Currently I have to scroll the list to check anyone's access.

**Comment by David-Fabijan** [comment]
Yes the user managament bit really needs a search and sort option. Right now I have 50 users and finding the right one to change say from Editor to Owner is a complete hassle.

**Comment by amandinedug** [comment]
Adding my use case to support this request: I manage documents with 500–700 users, and the lack of sorting affects two workflows:

- **Sharing panel** — updating a specific user's permissions is painful when you have to scroll through hundreds of unsorted names
- **"See As"** — verifying that ACL rules are correctly applied for a given user is equally frustrating, as the dropdown is unsorted too

I also opened a thread on the community forum if others want to weigh in: https://community.getgrist.com/t/feature-request-alphabetical-sorting-of-users-in-document/13916

Alphabetical sorting in both places would make a huge difference.
